packages/input-container/src/vaadin-input-container.js

```diff
@@ -19,6 +19,11 @@ export class InputContainer extends ThemableMixin(DirMixin(PolymerElement)) {
           display: flex;
           align-items: center;
           flex: 0 1 auto;
+          border-radius: var(--vaadin-input-container-top-left-radius, var(--vaadin-input-container-border-radius))
+            var(--vaadin-input-container-top-right-radius, var(--vaadin-input-container-border-radius))
+            var(--vaadin-input-container-bottom-right-radius, var(--vaadin-input-container-border-radius))
+            var(--vaadin-input-container-bottom-left-radius, var(--vaadin-input-container-border-radius));
+          --vaadin-input-container-border-radius: 0px;
```

Comment:
Let’s not define any custom properties on the host. This prevents developers from overriding them globally on the `html` level, which is most convenient.

```suggestion
          border-radius: var(--vaadin-input-container-top-left-radius, var(--vaadin-input-container-border-radius, 0px))
            var(--vaadin-input-container-top-right-radius, var(--vaadin-input-container-border-radius, 0px))
            var(--vaadin-input-container-bottom-right-radius, var(--vaadin-input-container-border-radius, 0px))
            var(--vaadin-input-container-bottom-left-radius, var(--vaadin-input-container-border-radius, 0px));
```

Reply:
Alternative way.

```suggestion
          border-radius: var(--vaadin-input-container-top-left-radius, var(--_border-radius))
            var(--vaadin-input-container-top-right-radius, var(--_border-radius))
            var(--vaadin-input-container-bottom-right-radius, var(--_border-radius))
            var(--vaadin-input-container-bottom-left-radius, var(--_border-radius));
          --_border-radius: var(--vaadin-input-container-border-radius, 0px);
```

Reply:
I’m also wondering if we should use logical props. But browser support begins only from Safari 15. So I suppose would need to do something like this:

```css
border-top-left-radius: var(--vaadin-input-container-start-start-radius);
border-start-start-radius: var(--vaadin-input-container-start-start-radius);
...
```

Reply:
Changed to use logical radius props.